app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -109,6 +109,23 @@
       .js-hidden {
         display: none;
       }
+
+      // stylelint-disable declaration-no-important
```

Comment:
You can add `// stylelint-disable-line declaration-no-important` at the end of specific lines, e.g.

```CSS
display: none !important; // stylelint-disable-line declaration-no-important
```

Reply:
The occurrence of `// stylelint-disable declaration-no-important` that appears in this file wraps a block of rules that include two instances of `!important`. In this case, I think it's fine.